RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -38,7 +38,7 @@ metsig::METSignificance::~METSignificance() {
 reco::METCovMatrix
 metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
 				       const std::vector< edm::Handle<reco::CandidateView> >& leptons,
-				       const edm::View<reco::Candidate>& pfCandidates,
+				       const edm::Handle<edm::View<reco::Candidate> >& pfCandidates,
```

Comment:
why was this change needed?
The current implementation does not use anything specific to the edm::Handle.
This implementation only carries an extra cost of dereferencing this handle all the time.